### PR TITLE
[thread-cert] wrap command `br peers` and `br routers`

### PR DESCRIPTION
**Pull request by zesonzhang**
This commit adds functions to read peer BRs and routers on infra link by wrapping the ot-ctl command `br peers` and `br routers`. 

`test_multi_ail.py` is also updated to test the new added functions.

### CODE REVIEW

File: tests/scripts/thread-cert/node.py [hunk 1]
```diff
@@ -4072,6 +4072,35 @@ def get_ip6_address(self, address_type: config.ADDRESS_TYPE):
...
+        #   fe80:0:0:0:42:acff:fe14:3 (M:0 O:0 Stub:1) ms-since-rx:144160 reachable:yes age:00:17:36 (peer BR)
+        #   fe80:0:0:0:42:acff:fe14:2 (M:0 O:0 Stub:1) ms-since-rx:45179 reachable:yes age:00:17:36
+        #   Done
+        self.send_command('br routers')
+        return self._expect_command_output()
+
+    def get_br_routers_ip_addresses(self) -> List[IPv6Address]:
+        """parse `br routers` output and return the list of IPv6 addresses"""
+        return [IPv6Address(line.split()[0]) for line in self.get_br_routers()]
+
```
Comment (abtink) [inline tests/scripts/thread-cert/node.py:4103]: Suggest moving these function close to other `br` related ones in this file?

We have `def enable_br(self):`, `def disable_br(self):`, `def get_br_omr_prefix(self)`, ...

**Review by abtink — APPROVED** [review]
LGTM. Thanks.
One small suggestion below:

File: tests/scripts/thread-cert/node.py (hunk 1 above)
Comment (zesonzhang) [reply tests/scripts/thread-cert/node.py:4103]: sg, thanks! I will update

Comment (zesonzhang) [reply tests/scripts/thread-cert/node.py]: update!

**Review by superwhd — APPROVED** [review]
LGTM 👍

**Review by jwhui — APPROVED** [review]

**Pull request merged**